from .conainer_error import ContainerError


class ResolverError(ContainerError):
    pass
